activation/poetlistener.go

```diff
@@ -81,7 +82,7 @@ func NewPoetListener(net service.Service, poetDb PoetValidatorPersistor, logger
 		Log:               logger,
 		net:               net,
 		poetDb:            poetDb,
-		poetProofMessages: net.RegisterGossipProtocol(PoetProofProtocol),
+		poetProofMessages: net.RegisterGossipProtocol(PoetProofProtocol, priorityq.Low),
```

Comment:
is it low priority to send poet-proof ? @barakshani, though during hare it's probably best to forward hare first.

Reply:
It is all relative. @barakshani revisit priority order.

Reply:
Poet proofs are not super urgent.
@gavraz please provide the priority list we agreed upon, if you'd like me to revisit